### ISSUE TITLE
Add a11y nav

### ISSUE THREAD

**Issue by schalkneethling** [mdn/yari#1513]
Yari is still missing the `a11y` nav at the head of the page.

![Screenshot 2020-10-22 at 04 39 22](https://user-images.githubusercontent.com/10350960/96817880-90749300-1420-11eb-8682-b41be8dbe14d.png)
